### DISCUSSION TITLE
Can I install JingOS on my device?

### DISCUSSION THREAD

**Discussion by PaultheFish** [JingOS-team/JingOS#89] [Q&A]
1. For JingOS x86 version, we have postponed the development. If you wish to test it on your device, please make sure your device has at least 2GiB of RAM and is able to run Ubuntu. Devices with a touch screen like Surface Pro are recommended. 

Be aware, the team will not maintain the x86 version.

2. For ARM version, currently it only runs on JingPad A1. We have released the source code for your information if you want to attempt porting JingOS ARM to other devices.

**Comment by FabioLolix** [comment]
Where can be found the .dsc, orig.tar.xz and orig.tar.xz of your packages?
